diff --git a/src/libs/next/config/define-config.ts b/src/libs/next/config/define-config.ts
index d6928a4cec..0000000000 100644
--- a/src/libs/next/config/define-config.ts
+++ b/src/libs/next/config/define-config.ts
@@ -350,0 +351,3 @@ export function defineConfig(config: CustomNextConfig) {
+      '@napi-rs/canvas',
+      '@napi-rs/canvas-linux-arm64-gnu',
+      '@napi-rs/canvas-linux-x64-gnu',
